controllers/apis/apis_suite_test.go

```diff
@@ -0,0 +1,20 @@
+package apis_test
```

Comment:
This naming is inconsistent with `apiforms_test`. The following would also be consistent with `controllers_test`: 

```suggestion
package apicontrollers_test
```

Also `apis` with an `s` is kind of strange. I am not sure if it's a word that should be allowed in plural forms...ever 😆

Reply:
@olivierobert updated the name on [f3d4e6b](https://github.com/carryall/go-google-scraper-challenge/pull/26/commits/f3d4e6bcc278fd0cab9ff0643ca7b5b95708225a) 😄